dbt-tests-adapter/dbt/tests/adapter/query_comment/test_query_comment.py

```diff
@@ -55,7 +55,7 @@ def test_matches_comment(self, project):
 class BaseMacroArgsQueryComments(BaseDefaultQueryComments):
     @pytest.fixture(scope="class")
     def get_package_version(self, project):
-        return import_module("." + project.adapter_type, "dbt.adapters").__version__.version
+        return import_module("dbt.version").__version__
```

Comment:
IMHO ensuring that versions matches isn't related to query comments and shouldn't be tested here. After decoupling dbt-core from adapters, it is just a wrong assumption. I suggest to just remove this test, similar change was accepted in dbt-core https://github.com/dbt-labs/dbt-core/pull/9861

Reply:
I tested this against dbt-bigquery locally and since it, core and dbt-adapters all have different versions I'm reasonably confident we're getting it right here.

Reply:
@damian3031 great PR, thanks!